modules/common-workflows/pages/workflow-removing-channel.adoc

```diff
@@ -0,0 +1,149 @@
+
+. <<preparing-to-remove-channel>>
+. <<removing-channels>>
+. <<cleaning-up-channel-packages>>
+
+
+[[preparing-to-remove-channel]]
+=== Preparing to remove a channel
+
+Before removing a channel, you must identify the label for the channel to be removed.
+It can be done via {webui} or the command line.
+
+[IMPORTANT]
+====
+* Be careful not to remove channels that systems are currently subscribed to, or that you are planning on subscribing a system to.
+* If there are systems that are currently subscribed to a channel that you want to delete, do not remove the channel until those systems are upgraded or unsubscribed.
+====
+
+
+==== Identifying Channel labels
+
+.Procedure: Identifying channel labels using {webui}
+[role=procedure]
+. In the SUSE Manager {webui}, in the sidebar navigate to menu:Software[Channel List > All]
+. This page displays the [label]``Channel Name``. If you select the blue link for a channel name, you will see the field for [literal]``Channel Label``.
+. Identify the channel, and child channels, to be removed.
+
+
+.Procedure: Identifying channel labels using the command line
+[role=procedure]
+
+. You can get a quick list of channels by running the following command:
++
+----
+spacewalk-remove-channel -l
+----
+
+
+====  Verify channel system subscriptions
+
+.Procedure: Verifying channel system subscriptions using {webui}
+
+. In the {productname} {webui}, in the sidebar navigate to menu:Software[Channel List > All].
+. Locate [literal]``Systems`` column on the right.
+. Check the [literal]``Systems`` column for the channel, or any child channels, that you want to remove.
+
+
+.Procedure: Verify Channel System Subscriptions using the command line
+
+. Run the following command:
++
+----
+spacecmd -- softwarechannel_listsystems <Channel Label>
+---- 
+
+
+[[removing-channel]]
+=== Removing channel
+
+A channel, with its metadata, can be removed in the SUMA command line by using the [literal]``spacewalk-remove-channel`` command. 
+Keep in mind that this command does not remove the packages themselves from the filesystem. 
+
+Removing packages from the filesystem is described in <<cleaning-up-channel-packages>>.
+
+
+
+==== Removing an individual channel
+
+.Procedure: Removing an individual channel using the command line
+
+. To remove an individual channel run the following command:
++
+----
+spacewalk-remove-channel -c channel-label
+----
++
+. To remove multiple channels at the same time, use [literal]``-c`` flag for each channel, followed by the [literal]``channel-label``. 
+  For example:
++
+----
+spacewalk-remove-channel -c channel-label1 -c channel-label2
+----
+
+
+==== Removing a Parent and all its Children
+
+.Procedure: Removing a Parent and all its Children
+. It's also possible to remove a parent channel with all of its children, like this:
++
+----
+spacewalk-remove-channel -a sles12-sp5-pool-x86_64
+----
+
+
+[[cleaning-up-channel-packages]]
+=== Cleaning up channel packages
+
+==== Running the sync
```

Comment:
Instead of "sync" better use synchronization.

Reply:
done, thanks